cdap-spark-core/src/main/java/co/cask/cdap/app/runtime/spark/SparkRuntimeService.java

```diff
@@ -362,15 +386,16 @@ public Void call() throws Exception {
       }
     }
 
-    String extraClassPath = "$PWD/" + CDAP_SPARK_JAR + "/lib/*";
+    String extraClassPath = "$PWD/" + CDAP_LAUNCHER_JAR + File.pathSeparator + "$PWD/" + CDAP_SPARK_JAR + "/lib/*";
```

Comment:
Probably not. Can change that to use `File.separate` to construct the path. Finding if there is better library for doing the job.